FUNCTION-SPEC.md

```diff
@@ -63,6 +63,27 @@ module.exports = lambdaCfn.build({
 ```
 - Comma separated parameter values can be parsed within the function code with the built-in `lambdaCfn.splitOnComma` function, which will return an array of values from a comma delimited string. If `process.env.someParameter` = `value1,value2,value3,value4` then `splitOnComma(process.env.someParameter)` returns `['value1','value2','value3','value4']`
 
+## Lambda alarms
+By default, CloudWatch alarms are configured to alarm after threshold of 1 or more errors happen over a period of 5 minutes. These values, `threshold`, `period`, and `evaluationPeriods` can be configured change how your function alarms. (For example, for non-critical stacks, setting the treshold to 3 will inform you that the [lambda could not do a successful retry](https://docs.aws.amazon.com/lambda/latest/dg/retries-on-errors.html).)
+
```

Comment:
I think we're fine linking to the CloudWatch documents here instead of recreating them in our docs. https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html#alarm-evaluation